src/SFA.DAS.AssessorService.Web/Controllers/SearchController.cs

```diff
@@ -98,7 +99,7 @@ private void GetSelectedStandardViewModel(SearchRequestViewModel result)
                 ShowExtraInfo = resultViewModel.ShowExtraInfo,
                 UlnAlreadyExists = resultViewModel.UlnAlreadyExists,
                 IsNoMatchingFamilyName = resultViewModel.IsNoMatchingFamilyName,
-                Versions = resultViewModel?.Versions.Select(v => (StandardVersion)v)
+                Versions = resultViewModel?.Versions
```

Comment:
can resultViewModel be null - if it can this check is good but why no check above?